doc/user_guide/ide-integration.rst

```diff
@@ -15,7 +15,7 @@ To use Pylint with:
  - WingIDE_, see https://wingware.com/doc/warnings/external-checkers,
  - PyCharm_, see :ref:`the section below <pylint_in_pycharm>`,
  - TextMate_, see :ref:`the section below <pylint_in_textmate>`
- - `Visual Studio Code`_, see https://docs.microsoft.com/en-us/visualstudio/python/linting-python-code,
+ - `Visual Studio Code`_, see https://code.visualstudio.com/docs/python/linting,
```

Comment:
```suggestion
 - `Visual Studio Code`_, see https://code.visualstudio.com/docs/python/linting,
 - `Visual Studio`_, see https://docs.microsoft.com/en-us/visualstudio/python/linting-python-code,
```

Reply:
```suggestion
 - `Visual Studio Code`_, see https://code.visualstudio.com/docs/python/linting,
  - `Visual Studio Code Pylint Extension`_ see, https://marketplace.visualstudio.com/items?itemName=ms-python.pylint,
 - `Visual Studio`_, see https://docs.microsoft.com/en-us/visualstudio/python/linting-python-code,
```

Let's also add a link to the new extension Microsoft is maintaining 😄